pkg/resource/user/sdk.go

```diff
@@ -262,10 +262,6 @@ func (rm *resourceManager) sdkCreate(
 	}
 
 	rm.setStatusDefaults(ko)
-	if err := rm.syncPolicies(ctx, &resource{ko}); err != nil {
-		return nil, err
-	}
```

Comment:
same comment as above... the user's policies will not be set until the user spec is updated or the 10 hour default requeue happens.